src/display/annotation_helper.js

```diff
@@ -217,6 +217,9 @@ var AnnotationUtils = (function AnnotationUtilsClosure() {
 
     var link = document.createElement('a');
     link.href = link.title = item.url || '';
+    if (PDFJS.openExternalLinksInNewWindow) {
```

Comment:
Won't this set `target = '_blank'` for both external *and* internal links, when the condition is `true`? You probably need to do something along these lines instead:
```javaScript
if (item.url && PDFJS.openExternalLinksInNewWindow) {
```